src/VerifyHandler.php

```diff
@@ -132,7 +136,7 @@ public function getComponent(): string
      */
     protected function getCredentialRequestOptions(
         StoreInterface $store,
-        RegisteredMethod $registeredMethod,
+        RegisteredMethod $registeredMethod = null,
```

Comment:
Doc block could use a tweak for being nullable and the new AuthenticationFailedException